source/Calamari.Aws/Integration/CloudFormation/CloudFormationObjectExtensions.cs

```diff
@@ -276,13 +335,13 @@ public static Task<DeleteStackResponse> DeleteStackAsync(this Func<IAmazonCloudF
         {
             try
             {
-                var response =  await clientFactory().CreateStackAsync(request);
+                var response = await clientFactory().CreateStackAsync(request);
                 return response.StackId;
             }
             catch (AmazonCloudFormationException ex) when (ex.ErrorCode == "AccessDenied")
             {
                 throw new PermissionException(
-                    "The AWS account used to perform the operation does not have the required permissions to create the stack.\n"+
+                    "The AWS account used to perform the operation does not have the required permissions to create the stack.\n" +
```

Comment:
Why `\n`? Can't we just let the UI deal with it?

Reply:
I'll remove